components/abstractions/src/main/java/com/microsoft/kiota/RequestAdapter.java

```diff
@@ -32,9 +32,12 @@ public interface RequestAdapter {
      * @param <ModelType> the type of the response model to deserialize the response into.
      * @return the deserialized response model.
      */
-    @Nullable
-    @SuppressWarnings("LambdaLast")
-    <ModelType extends Parsable> ModelType send(@Nonnull final RequestInformation requestInfo, @Nonnull final ParsableFactory<ModelType> factory, @Nullable final HashMap<String, ParsableFactory<? extends Parsable>> errorMappings);
+    @Nullable @SuppressWarnings("LambdaLast")
```

Comment:
same thing here, I'm not a big fan of that change either. But I might be biased by dotnet.

Reply:
No strong opinions here, I let you take the lead, I'm happy with whatever format as long as it's automated.

Reply:
(fixes the configuration for SuppressWarnings, resolving)